packages/docs/content/04-clusters/1-new-clusters.md

```diff
@@ -2014,86 +2268,97 @@ By default, 4GB of memory is allocated for private gateways. Please ensure that
 
-``` 
 ## Tenant Portal - Launch Cloud Gateway
-* Close the 'Create New Gateway' dialogue, if still open, or navigate to the Private Cloud Gateway page under settings in case you have navigated away or been logged out.
-* Wait for a gateway widget to be displayed on the page and for the "Configure" option to be available.This may take few minutes.
-* Click on the "Configure" button to invoke the Spectro Cloud Configuration dialogue. Provide the OpenStack credentials and proceed to the next configuration step.
-
-|Parameter      | Value      | Remark           | 
-|:-------------|:----------|:--------------------
-|Cloud Account Name| Custom name for the cloud account | The name will be used to identify the gateway instance. Typical environments may only require a single gateway to be set up, however, multiple gateways might be required for managing clusters. Choose a name that can easily identify the environment that this gateway instance is being configured for.
-|User Name| OpenStack User Name| |
-| Password| OpenStack Password| |
-|Identity Endpoint| OpenStack identity URL |URL to the management platform portal|
-|Use self-signed certificate| | |
-| Parent Region | The value obtained while creating an OpenStack account.| Obtained while creating OpenStack account|
-|Default Domain| An Identity API v3 entity | An Identity API v3 entity|
-|Default Project(Optional)|Represent the base unit of “ownership” in OpenStack|An isolated group of zero or more users who share common access with specific privileges.|
-|Share account across platform projects|Check/Uncheck |
-
-
-
-* Verify these credentials and proceed to the next level of deploying the gateway installer. Fill in the following parameters to the deployment wizard.
-
-|Parameter      | Value      | Remark           | 
-|:-------------|:----------|:--------------------
-|SSH Key Name| The key, useful for troubleshooting purposes (Recommended)|Enables SSH access to the VM as 'ubuntu' user|
-|Placement|Dynamic/Static |  |
-|Node CIDR|Optional - IP range exclusive to Nodes|This range should be different to prevent an overlap with your network CIDR|
-|DNS Name Server (optional)|8.8.8.8 default|A system by which Internet domain name-to-address and address-to-name resolutions are determined|
-|Availability Zones|Make the choices from the list. |Logical abstraction for partitioning an OpenStack cloud for users|
-|Flavor| Select from the drop-down list|Defines the compute, memory, and storage capacity of nova computing instances|
-|Number of nodes|1 or 3 | |
+* Navigate back to the Private Cloud Gateway page in tenant console under settings.
+* The deployment progress can be viewed in the tenant console. This will take few minutes
+* Once the deployment is  completed, set the number of nodes as either 1 or 3.
```

Comment:
Not required. As we have already set. Remove this point "Once the deployment is completed, set the number of nodes as either 1 or 3."